classes/dbutils.js

```diff
@@ -3,9 +3,10 @@ const {
     LLR
 } = require("./utilities.js");
 
-async function get_matches(db, limit) {
+async function get_matches(db, { limit = 100, network } = {}) {
```

Comment:
What's the benefit using a dict instead of plain parameters like `get_matches(db, network, limit=100)`?